packages/apptive_grid_core/lib/src/model/form/form_block.dart

```diff
@@ -0,0 +1,78 @@
+/// A freeform block inside of a form
+class FormBlock {
+  /// Deserializes [json] into a FormBlock Object
+  factory FormBlock.fromJson(dynamic json) => FormBlock(
+        id: json['id'],
+        text: json['text'],
+        fieldIndex: json['fieldIndex'],
+        type: json['type'],
+        style: json['style'],
+        pageId: json['pageId'],
+      );
+
+  /// Creates a FormBlock Object
+  FormBlock({
+    required this.id,
+    required this.pageId,
+    required this.fieldIndex,
+    required this.text,
+    required this.type,
+    required this.style,
+  });
+
+  /// Id of the block
+  final String id;
+
+  /// Page of the block in the form
+  final String pageId;
+
+  /// Position of the block in the page
+  final int fieldIndex;
+
+  /// Text of the block
+  final String text;
+
+  /// Type of the block
+  final String type;
+
+  /// Style of the block
+  final String style;
```

Comment:
I don't see a usage of these yet in the Forms package yet. I agree that this should already be parsed to avoid having to add this to the core later however I think right now the documentation should be extended to explain a bit more what this actually is used for (from the apptivegrid side of things)

Reply:
done